gimme_aws_creds/main.py

```diff
@@ -549,6 +557,9 @@ def okta(self):
         if self.conf_dict.get('preferred_mfa_type'):
             okta.set_preferred_mfa_type(self.conf_dict['preferred_mfa_type'])
 
+        if self.conf_dict.get('preffered_mfa_provider'):
```

Comment:
I added `preffered_mfa_provider` option in `okta_aws_login_config`. This will automatically select the specific mfa provider type like `GOOGLE`.